Description: The new version of Xlib.h has changed the name of the
 _XLIB_H_ macro to _X11_XLIB_H_ (debian package: libx11-dev 2:1.4.1-4,
 upstreamer: http://go.debian.net/Sksv). This patch changes image.h in
 order to be able to use both identifiers.
Forwarded: no
Origin: other, http://bugs.gentoo.org/344993
Bug-Debian: http://bugs.debian.org/614484
Last-Update: 2011-02-27

--- a/image.h
+++ b/image.h
@@ -63,7 +63,7 @@
     unsigned long sourceMask;
 } Image;
 
-#ifdef _XLIB_H_
+#if defined(_XLIB_H_) || defined(_X11_XLIB_H_)
 /* Used to transfer information to routines in iprocess.c */
 struct imageprocessinfo {
     int oilArea;
